test/vscontext.mock.ts

```diff
@@ -10,6 +10,9 @@ class DummyMemento implements vscode.Memento {
     dummyMomentoData[key] = value;
     return Promise.resolve(dummyMomentoData);
   }
+  public setKeysForSync(keys: string[]): void {
```

Comment:
Is this the side-effect of bumping engine to ^1.50.0

Reply:
Yes, this has been added to globalState ^1.50.0

Reply:
to get it in sync with https://github.com/redhat-developer/vscode-commons/blob/master/package.json#L17